scripts/split-paired-reads.py

```diff
@@ -128,47 +131,48 @@ def main():
         # Use default filename created above
         fp_out2 = get_file_writer(open(out2, 'wb'), args.gzip, args.bzip)
 
+    # put orphaned reads here, if -0!
+    allow_orphans = False
```

Comment:
On Tue, Aug 04, 2015 at 03:54:24PM -0700, Michael R. Crusoe wrote:
> > @@ -128,47 +131,48 @@ def main():
> >          # Use default filename created above
> >          fp_out2 = get_file_writer(open(out2, 'wb'), args.gzip, args.bzip)
> >  
> > +    # put orphaned reads here, if -0!
> > +    allow_orphans = False
> 
> I'm okay with `not args.output_orphaned` 

That doesn't address "here's my reasoning, what do you think" but I'll give
it a whirl ;)